staging/provisioning/windows/windowssecuretls.ps1

```diff
@@ -0,0 +1,387 @@
+﻿#***************************************************************************************************************
+# Below functions are modified from TLSSettings.ps1
+# It does the following:
+#   *   Disable TLS 1.O, TLS 1.1, SSLv2, SSLv3 and enables TLS1.2
+#   *   The CipherSuite order is set to the SDL approved version.
+#   *   The FIPS MinEncryptionLevel is set to 3.
+#   *   RC4 is disabled
+#***************************************************************************************************************
+
+#******************* FUNCTION THAT ACTUALLY UPDATES KEYS; WILL RETURN REBOOT FLAG IF CHANGES ***********************

```

Comment:
Is this comment accurate? 
I don't see where a flag that indicates if a reboot needed gets returned.

Reply:
I removed the restart since we always restart the Windows node. Will update the comment.